scrapy/extensions/robotstxtparser.py

```diff
@@ -0,0 +1,137 @@
+import sys
+import logging
+
+from scrapy.utils.python import to_native_str, to_unicode
+
+logger = logging.getLogger(__name__)
+
+class PythonRobotParser():
+    def __init__(self, content, spider):
```

Comment:
What doyou think about renaming `content` variable name to `robotstxt_body`, or something like that, for clarity?